draft-ietf-dnsop-svcb-https.md

```diff
@@ -1252,8 +1274,9 @@ be populated with the registrations below:
 | 2           | no-default-alpn | No support for default protocol | (This document) |
 | 3           | port            | Port for alternative service    | (This document) |
 | 4           | ipv4hint        | IPv4 address hints              | (This document) |
-| 5           | echconfig      | Encrypted ClientHello info      | (This document) |
+| 5           | echconfig       | Encrypted ClientHello info      | (This document) |
 | 6           | ipv6hint        | IPv6 address hints              | (This document) |
+| 7           | mandatory       | Essential keys in this RR       | (This document) |
```

Comment:
Did we want to give it ID 0 ?  (I liked the idea of having it first.)  I guess we can't make it 1 because we don't want to move alpn?

Reply:
I’d rather us not move alpn if that’s okay :)

Any reason not to give this 0? It is a meta-key, so making it 0 seems elegant.

Reply:
We'd previously discussed leaving 0 unassigned so that APIs could use it to mean something like "null".  If we don't think that's actually useful then I'm OK with making this 0.

Reply:
I changed "mandatory" to ID 0 and made key65535 (a.k.a. -1) explicitly reserved as "invalid key".